libs/category/driver/magento/src/category.service.ts

```diff
@@ -153,4 +161,8 @@ export class DaffMagentoCategoryService implements DaffCategoryServiceInterface
 
     return this.magentoCategoryResponseTransformer.transform(completeCategory);
   }
+
+  private truncateUriExtension(uri: string): string {
```

Comment:
We have the prop `truncateUri` in the configuration, but it appears unused.

Reply:
good call, fixed